src/Integration.Vsix.UnitTests/Analysis/AnalysisConfigMonitorTests.cs

```diff
@@ -66,6 +92,18 @@ public void WhenSupressionsUpdated_AnalysisIsRequested()
             builder.Logger.AssertOutputStringExists(AnalysisStrings.ConfigMonitor_SuppressionsUpdated);
         }
 
+        [TestMethod]
+        public void WhenSuppressionsUpdated_HasSubscribersToConfigChangedEvent_SubscribersNotified()
```

Comment:
I hadn't thought about suppressions; strictly speaking we don't need to re-calculate the effective rules or call init-linter in this case, but it won't happen often so it won't matter too much.

Reply:
It would happen if it's a proj in connected mode that also happens to have JS files, but yeah it would be every 10 min + on binding update.